content/automate/ManageIQ/System/Event/EmsEvent/Hawkular.class/hawkular_datasource.error.yaml

```diff
@@ -0,0 +1,14 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name: Datasource Error
+    name: hawkular_datasource.error
+    inherits:
+    description:
+  fields:
+  - rel4:
+      value: "/System/event_handlers/event_action_refresh?target=ems"
+  - rel5:
+      value: "/System/event_handlers/event_action_policy?target=middleware_server&policy_event=mw_datasource_error&param="
```

Comment:
@tumido ```policy_event``` raised in this PR does not match what is defined in [your PR for MiqEventDefinition](https://github.com/ManageIQ/manageiq/pull/16134/files#diff-da17f6bf8d1c54c7878221e8110a0577R77).